sycl/include/sycl/exception_list.hpp

```diff
@@ -52,5 +53,23 @@ class __SYCL_EXPORT exception_list {
 
 using async_handler = std::function<void(sycl::exception_list)>;
 
+namespace detail {
+// Default implementation of async_handler used by queue and context when no
+// user-defined async_handler is specified.
+inline void defaultAsyncHandler(exception_list Exceptions) {
+  std::cout << "Default async_handler caught exceptions:";
```

Comment:
```suggestion
  std::cerr << "Default async_handler caught exceptions:";
```
?

Reply:
`std::cerr` seems more fitting indeed! It has been changed.